src/plugins/data_views/common/data_views/data_view.ts

```diff
@@ -493,6 +493,8 @@ export class DataView extends AbstractDataView implements DataViewBase {
         aggregatable: true,
         searchable: true,
         count: config.popularity ?? 0,
+        customLabel: config.customLabel,
+        customDescription: config.customDescription,
```

Comment:
Were they missing here or it's expected?

Reply:
These are applied a few lines lower. It looks like setting `count` here can be removed since it happens a few lines lower as well.